azure-pipelines-arcade.yml

```diff
@@ -273,6 +273,8 @@ stages:
 - ${{ if eq(variables._RunAsInternal, True) }}:
   - template: eng\common\templates\post-build\post-build.yml
     parameters:
+      variables:
+        - group: SDL_Settings
```

Comment:
This variable group has values specific to Arcade, please do not use this. Create one of your own.

Reply:
And variable group needs to be added here - https://github.com/dotnet/wcf/blob/9c7d5d85d7466871f4c8d2d85473301443e48564/azure-pipelines-arcade.yml#L34

Reply:
Thanks for your assistance @sunandabalu 
Added a Wcf group for the SDL validation params and amended the commit.